apps/integration/integration/queries/options.py

```diff
@@ -0,0 +1,49 @@
+from collections.abc import Iterable
+
+
+class Option:
+    def __init__(self, name, values):
+        if isinstance(values, str) or not isinstance(values, Iterable):
+            values = [values]
+        if not isinstance(values, list):
+            values = list(values)
+        self._name = name
+        self._values = values
+        self._current_pos = 0
+
+    def get(self):
+        return self._values[self._current_pos]
+
+    def get_name(self):
+        return self._name
+
+    def advance(self):
+        self._current_pos += 1
+        if self._current_pos == len(self._values):
+            self._current_pos = 0
+            return True
+        return False
```

Comment:
Nit : After starring at it for a while, I understand how it is being used but ideally, Would you not want to return True if an option can advance?

Reply:
True means the next thing can advance. Think of the return value as a carry bit in a counter, cuz that's essentially what it represents